[
  {
    "type": "feature",
    "category": "",
    "description": "Retry on a set of CURLE_*_ERROR based failures. Currently, only CURLE_RECV_ERROR (errno 56) is retried."
  }
]